### PR TITLE
Create B827EBFFFF46EC6D.json

### PR DESCRIPTION
**Pull request by mftutui**

### CODE REVIEW

File: B827EBFFFF46EC6D.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "gateway_ID": "B827EBFFFF46EC6D",
+    "servers": [
+      {
+        "server_address": "router.us.thethings.network",
+        "serv_port_up": 1700,
+        "serv_port_down": 1700,
+        "serv_enabled": true
+      }
+    ],
+    "ref_latitude": "-27.571021",
```
Comment (gonzalocasas) [inline B827EBFFFF46EC6D.json]: The numbers should be numbers, not strings.
```suggestion
    "ref_latitude": -27.571021,
```

File: B827EBFFFF46EC6D.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "ref_longitude": "-48.508433",
```
Comment (gonzalocasas) [inline B827EBFFFF46EC6D.json]:
```suggestion
    "ref_longitude": -48.508433,
```

File: B827EBFFFF46EC6D.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    "ref_altitude": "",
```
Comment (gonzalocasas) [inline B827EBFFFF46EC6D.json]: this will probably break, use a real altitude or at least, zero (as number, not string).

Comment (mftutui) [reply B827EBFFFF46EC6D.json]: Thank you!

**Comment by gonzalocasas** [comment]
Thanks! merged!

**Pull request merged**